test/remote/gateways/remote_authorize_net_test.rb

```diff
@@ -384,10 +384,20 @@ def test_successful_authorization_with_moto_retail_type
   end
 
   def test_successful_verify
+    card_option = {
+      brand: 'jcb'
+    }
+    other_credit_card = credit_card('3088000000000017', card_option)
+    response = @gateway.verify(other_credit_card, @options)
+    assert_success response
+    assert_equal 'This transaction has been approved', response.message
+    assert_equal response.responses.count, 2
```

Comment:
💬  Comment:

Pleae remember to fix that test that we saw is using response. responses.last, adding unneeded complexity to that assertion.

Reply:
The test have been modified, I change the assert to check the number of responses in the scenario of one dollar auth, thinking about both operations, auth and void, the responses.last is not used anymore.